codespell_lib/tests/test_basic.py

```diff
@@ -391,6 +391,65 @@ def test_ignore_word_list(
     assert cs.main("-Labandonned,someword", "-Labilty", tmp_path) == 1
 
 
+@pytest.mark.parametrize(
+    ("content", "expected_error_count"),
+    [
+        # recommended form
+        ("abandonned abondon abilty  # codespell:ignore abondon", 2),
+        ("abandonned abondon abilty  // codespell:ignore abondon,abilty", 1),
+        (
+            "abandonned abondon abilty  /* codespell:ignore abandonned,abondon,abilty",
+            0,
+        ),
+        # ignore unused ignore
+        ("abandonned abondon abilty  # codespell:ignore nomenklatur", 3),
+        # wildcard form
+        ("abandonned abondon abilty  # codespell:ignore ", 0),
+        ("abandonned abondon abilty  # codespell:ignore", 0),
+        ("abandonned abondon abilty  # codespell:ignore\n", 0),
+        ("abandonned abondon abilty  # codespell:ignore\r\n", 0),
+        ("abandonned abondon abilty  # codespell:ignore  # noqa: E501\n", 0),
+        ("abandonned abondon abilty  # codespell:ignore # noqa: E501\n", 0),
+        ("abandonned abondon abilty  # codespell:ignore# noqa: E501\n", 0),
+        ("abandonned abondon abilty  # codespell:ignore, noqa: E501\n", 0),
+        ("abandonned abondon abilty  #codespell:ignore\n", 0),
+        # ignore these for safety
+        ("abandonned abondon abilty  # codespell:ignorenoqa: E501\n", 3),
+        ("abandonned abondon abilty  codespell:ignore\n", 3),
+        ("abandonned abondon abilty codespell:ignore\n", 3),
+        # showcase different comment markers
```

Comment:
Just testing another edge case explicitly
```suggestion
        # ignore these as they aren't valid
        ("abandonned abondon abilty  # codespell:igore\n", 3),
        # showcase different comment markers
```